docs/index.rst

```diff
@@ -77,7 +77,7 @@ When pgbedrock is run, it would make sure that:
     * ``jdoe`` does not have any access other than that listed above (except whatever it inherits
       from the ``analyst`` role that ``jdoe`` is a member of)
 
-    .. _pg_authid: https://www.postgresql.org/docs/9.1/static/catalog-pg-authid.html
+    .. _pg_authid: https://www.postgresql.org/docs/9.6/static/catalog-pg-authid.html
```

Comment:
silly question, but does pgbedrock support pg10?

Reply:
@cpdean informed me that no, it does not.

Reply:
That's actually a very good question. So I _think_ that pgbedrock itself supports pg10 but our test suite does not: Postgres 10 introduced a few roles that our test suite doesn't know about, so it fails saying that there are roles missing / present that it doesn't expect. That's actually a reason I may hold off on merging this PR: I'd like to get Postgres 10 support in here if it is as simple as spending a half hour fixing a few tests, then add it to the badge.

I may have some time some night in the next few days to tinker, so I'll at least hold off on merging this PR until I've investigated a bit more.